### PR TITLE
Support new activation layer "ceil" for VPU

### PR DESCRIPTION
**Pull request by andrejsokolov**
Add new Operation "Ceiling" for VPU Myriad
task: -42885

### CODE REVIEW

File: inference-engine/src/vpu/graph_transformer/src/stages/ceiling.cpp [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <vpu/frontend/frontend.hpp>
+
+#include <vector>
+#include <memory>
+#include <set>
+
+#include <vpu/stages/post_op_stage.hpp>
```
Comment (AndrewBakalinIntel) [inline inference-engine/src/vpu/graph_transformer/src/stages/ceiling.cpp:11]: Nitpick: better to place it before C++ libraries accroding to the code style.

File: inference-engine/src/vpu/graph_transformer/src/stages/ceiling.cpp [hunk 2]
```diff
@@ -0,0 +1,44 @@
+// Copyright (C) 2018-2020 Intel Corporation
```
Comment (AndrewBakalinIntel) [inline inference-engine/src/vpu/graph_transformer/src/stages/ceiling.cpp:1]: Nit: I think `2020` only should be here.

**Review by AndrewBakalinIntel — APPROVED** [review]
LGTM in general, just few minor comments left.

File: inference-engine/src/vpu/graph_transformer/src/stages/ceiling.cpp (hunk 1 above)
Comment (andrejsokolov) [reply inference-engine/src/vpu/graph_transformer/src/stages/ceiling.cpp]: Fixed

File: inference-engine/src/vpu/graph_transformer/src/stages/ceiling.cpp (hunk 2 above)
Comment (andrejsokolov) [reply inference-engine/src/vpu/graph_transformer/src/stages/ceiling.cpp]: Fixed

**Comment by andrejsokolov** [comment]
@Maxim-Doronin, could you please review the PR?

**Review by Maxim-Doronin — APPROVED** [review]

**Review by iefode — APPROVED** [review]

**Pull request merged**